code/modules/antagonists/heretic/items/heretic_blades.dm

```diff
@@ -21,6 +21,13 @@
 	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "rend")
 	var/after_use_message = ""
 
+/obj/item/melee/sickly_blade/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/butchering, \
```

Comment:
With the new code you added, is this needed still?
It should be automatically applied now, and the vars are just the default so it seems like this just isn't needed

Reply:
I tested it out, and you are correct, it still properly butchers without the new code.